underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -50,9 +51,6 @@
 public final class GCSUnderFileSystem extends UnderFileSystem {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
-  /** Suffix for an empty file to flag it as a directory. */
-  private static final String FOLDER_SUFFIX = "_$folder$";
```

Comment:
I incline to still leave the `FOLDER_SUFFIX` still in each particular `XXXUnderFileSystem` class. The reason is we may have different suffix across different under file systems.  e.g., SWIFT has `/` instead of `_$folder$`